analysis/study_definition_prevax_sensitivity.py

```diff
@@ -23,7 +21,7 @@
 
 import json
 
-input_file = "output/input_vax_stage1_hosp.csv.gz"
+input_file = "output/input_prevax_stage1.csv.gz"
```

Comment:
This is causing an error for me as the YAML saves `output/input_prevax_stage1.rds` only so it cannot find `output/input_prevax_stage1.csv.gz`.

Reply:
created conversion script and actions to convert from rds to csv.gz as python study definition doesn't read rds